src/index.js

```diff
@@ -1,14 +1,22 @@
-var noop = function(){};
-var defaults = require('lodash.defaults');
-var isEmpty = require('lodash.isempty');
-var isFunction = require('lodash.isfunction');
-var isArray = require('lodash.isarray');
-var cloneDeep = require('lodash.clonedeep');
-var BrowserWindow = require('electron').BrowserWindow;
-var Menu = require('electron').Menu;
+const noop = function(){};
+const defaults = require('lodash.defaults');
+const isEmpty = require('lodash.isempty');
+const isFunction = require('lodash.isfunction');
+const isArray = require('lodash.isarray');
+const cloneDeep = require('lodash.clonedeep');
+const electron = require('electron');
+const remote = electron.remote;
+const Menu = electron.Menu || remote.Menu;
+const webContents = electron.webContents;
 
+// this allows us to support users who remote.require this or users who run it
+// directly in the renderer
+const currentWebContents = (process.type === 'browser' ?
```

Comment:
I think we should fetch the web contents on-demand—fetching them at the top level like this will cache them for all uses of `electron-editor-context-menu` in different windows/webviews, no?